Gruntfile.js

```diff
@@ -13,19 +13,20 @@ module.exports = function( grunt ) {
 	}
 
 	const fs = require( "fs" );
+	const { spawn } = require( "child_process" );
 	const gzip = require( "gzip-js" );
-	const nodeV14OrNewer = !/^v1[0-3]\./.test( process.version );
+	const nodeV16OrNewer = !/^v1[0-5]\./.test( process.version );
 	const nodeV17OrNewer = !/^v1[0-6]\./.test( process.version );
 	const customBrowsers = process.env.BROWSERS && process.env.BROWSERS.split( "," );
 
 	// Support: Node.js <14
 	// Skip running tasks that dropped support for Node.js 10 or 12
```

Comment:
Presumably you want to update this comment.

Node 17 has been EOL for over a year. Node 16 is EOL in Sept 2023.

Reply:
We still need to handle Node 10 as that's what our Jenkins uses and we cannot easily update it. We may get rid of Jenkins in favor of GitHub Actions but there are some complications with that. For now, jQuery needs to be able to build under Node 10.

Reply:
I meant that the comment doesn't match the variable names. I assumed you forgot to update the comment to correspond.

Reply:
Ah, nice catch!

Reply:
Updated.